packages/coreutils/src/url.ts

```diff
@@ -146,7 +148,10 @@ export namespace URLExt {
   export function isLocal(url: string): boolean {
     const { protocol } = parse(url);
 
-    return url.toLowerCase().indexOf(protocol) !== 0 && url.indexOf('/') !== 0;
+    return (
+      (!protocol || url.toLowerCase().indexOf(protocol) !== 0) &&
+      url.indexOf('/') !== 0
```

Comment:
I had to think hard about this, so please review carefully...

Reply:
👍